Peter Han JAVA Intern challenge solution - hanzhaolong0410@gmail.com

peterhan7 opened the pull request.

doublederek commented:
Hi there, I've updated the test file to reflect the proper number output, please pull down the changes and make any updates your PR as needed. Leave a comment when you do and I'll run the test. Thanks!

peterhan7 commented:
> Hi there, I've updated the test file to reflect the proper number output, please pull down the changes and make any updates your PR as needed. Leave a comment when you do and I'll run the test. Thanks!

Hi Derek, I have updated my branch against the upstream branch correctly. Please run the test. Thanks :)
